source/common/quic/envoy_quic_utils.cc

```diff
@@ -95,7 +95,7 @@ quic::QuicRstStreamErrorCode envoyResetReasonToQuicRstError(Http::StreamResetRea
   case Http::StreamResetReason::LocalReset:
     return quic::QUIC_STREAM_REQUEST_REJECTED;
   case Http::StreamResetReason::OverloadManager:
-    return quic::QUIC_STREAM_CANCELLED;
+    return quic::QUIC_STREAM_EXCESSIVE_LOAD;
```

Comment:
The comment for this error code in quic_error_codes.h says

```
// Peer exhibits a behavior that might be generating excessive load.
QUIC_STREAM_EXCESSIVE_LOAD = 24,
```

So it was originally for a different purpose.  Can we update that comment in QUICHE as a follow up?

Reply:
Thanks Bin!  Will do that in a QUICHE CL.